railties/lib/rails/application/configuration.rb

```diff
@@ -590,6 +592,12 @@ def respond_to_missing?(symbol, *)
       end
 
       private
+        def code_statistics_defaults
+          {
+            directories: CodeStatistics::DIRECTORIES
```

Comment:
We can expose CodeStatistics::TEST_TYPES (which is also used by third party gems) as well:
```suggestion
            directories: CodeStatistics::DIRECTORIES,
            test_types: CodeStatistics::TEST_TYPES
```